source/includes/fact-automatic-objectid.rst

```diff
@@ -0,0 +1,3 @@
+If you do not provide an :manual:`ObjectId
+</reference/bson-types/#objectid> in your document, |compass-short|
```

Comment:
Hey @jocelyn-mendez, check this out in staging; it's not rendering correctly. You're missing a closing backtick. See this suggestion:

```suggestion
</reference/bson-types/#objectid>` in your document, |compass-short|
```

Reply:
@jeff-allen-mongo Thanks just noticed it!